### PR TITLE
Polisen

### PR DESCRIPTION
**Pull request by antoncarlsson**

### CODE REVIEW

File: polisen/src/app.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require('dotenv').config();
+const app = require('express')();
+const bodyParser = require('body-parser');
+const { dateIsGiven, getDateRange } = './helpers/misc';
+const { createNewsRequestsFromDates, sendNewsRequests } = './helpers/news';
+
+const { APP_PORT } = process.env;
+app.use(bodyParser.json({ extended: true }));
+
+app.post('/api/polisens_nyheter', (req, res) => {
```
Comment (falapen) [inline polisen/src/app.js:10]: No error handling whatsoever (maybe add try-catch around object deconstruction)

File: polisen/src/helpers/misc.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const dateIsGiven = (body) => {
+  return (body !== undefined && body.from !== undefined && body.from !== '' && body.until !== undefined);
+};
+
+const stripTimeOfDate = (date) => date.substr(0, 10);
+
+const getDateRange = (from, until) => {
+  const dates = [];
+  for (const date = new Date(until); date >= new Date(from); date.setDate(date.getDate() - 1)) {
+    dates.push(stripTimeOfDate(date.toISOString()));
```
Comment (falapen) [inline polisen/src/helpers/misc.js]: Shouldn''t strip time

**Review by falapen — CHANGES_REQUESTED** [review]
Good in general. Should probably stop stripping time from the timespan.

File: polisen/src/app.js (hunk 1 above)
Comment (antoncarlsson) [reply polisen/src/app.js:10]: Added try-catch block.

**Review by falapen — APPROVED** [review]

**Comment by falapen** [comment]
Looks good

**Pull request merged**